lighthouse-core/lib/lh-error.js

```diff
@@ -7,21 +7,40 @@
 
 const i18n = require('./i18n/i18n.js');
 
+// TODO(exterkamp): Get better examples.
```

Comment:
remove
```suggestion
```

Reply:
https://github.com/GoogleChrome/lighthouse/pull/9114#issuecomment-509532813

Reply:
these still aren't checked but part of this PR...whats the plan with them?